src/ex6/client/src/components/TodoItem.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { MdOutlineModeEdit } from "react-icons/md";
+import { TiDeleteOutline } from "react-icons/ti";
+// eslint-disable-next-line
+import MondayModal from "./MondayModal";
+
+import { removeTodo, changeTodoStatus } from "../todos-client";
+import { MAX_TEXT_LENGTH } from "../constants";
+import styles from "../style/TodosItem.module.css";
+function TodoItem({
+  todoName = "no name was found",
+  todoStatus,
+  todoId,
+  onItemChange,
+}) {
+  const [status, setStatus] = useState(todoStatus);
+
+  const deleteTodo = async () => {
+    await removeTodo(todoId);
+    onItemChange();
+  };
+
+  const changeStatus = async () => {
+    await changeTodoStatus(todoId, !status);
+    onItemChange();
+  };
+  let itemName =
+    todoName.length > MAX_TEXT_LENGTH
+      ? todoName.substring(0, MAX_TEXT_LENGTH)
+      : todoName;
```

Comment:
you can also cut a text with css, by using max width . this way its according to the actualy size and t also adds a "..." so the user understand that the text is being cut

 ```width: 200px;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
```